prow/plugins/trigger/generic-comment.go

```diff
@@ -183,3 +231,47 @@ func getContexts(combinedStatus *github.CombinedStatus) (sets.String, sets.Strin
 	}
 	return failedContexts, allContexts
 }
+
+func markdownList(names []string) string {
```

Comment:
nit: inline this if you have but one caller, unless a standalone function is useful for testing

Reply:
Thanks. Inlined.